packages/docz/src/components/Docs.tsx

```diff
@@ -8,12 +8,60 @@ import { configSelector } from './ThemeConfig'
 
 export const isFn = (value: any): boolean => typeof value === 'function'
 
-const sortBy = (a: any, b: any, reverse?: boolean) => {
+const compare = (a: any, b: any, reverse?: boolean) => {
   if (a < b) return reverse ? 1 : -1
   if (a > b) return reverse ? -1 : 1
   return 0
 }
 
+const UNKNOWN_POS = 999999  // TODO: fix this?
```

Comment:
I was thinking to try something like that:
https://github.com/mdx-js/mdx/blob/master/docs/_app.js#L49-L60

What do you think about @mpivaa ?

Reply:
I thought about `Infinity` too, I will change